tests/integration/test_fim/test_files/test_audit/test_audit.py

```diff
@@ -301,7 +301,15 @@ def get_audit_creation_time():
                 return proc.create_time()
         pytest.fail("Auditd is not running")
 
-    plugin_path = "/etc/audisp/plugins.d/af_wazuh.conf"
+    audisp_path = "/etc/audisp/plugins.d/af_wazuh.conf"
+    audit_path = "/etc/audit/plugins.d/af_wazuh.conf"
+
+    if os.path.exists(audisp_path):
+        plugin_path = audisp_path
+    elif os.path.exists(audit_path):
+        plugin_path = audit_path
+    else:
+        raise Exception('Audit plugin not found')
```

Comment:
This will produce an error instead of failing in tests execution. Every test should handle precisely every case and ensure every time that it fails is due to the product instead of the environment or the test itself.

It should be desirable to investigate what should be the Wazuh log output if Audit plugins are not found

Reply:
Fixed in 9758990